### ISSUE TITLE
Unhandled error 'Request Entity Too Large'

### ISSUE THREAD

**Issue by mjmaix** [keystonejs/keystone#522]
#422 adds the ability to set a file size limit but the exception thrown is not handled properly.

Upload is stopped but the page rendered shows like this:
![image](https://cloud.githubusercontent.com/assets/1903059/3866659/0cf62576-1fde-11e4-9e54-a3158554fbaa.png)
Server error: `Error: Request Entity Too Large
  at Object.exports.error `
It would be better if this is handled by `req.flash` error

**Comment by creynders** [comment]
Closing due to inactivity, feel free to comment again if this issue persists for you!

**Comment by AndKenneth** [comment]
If anyone else is looking as to how to increase the file size limit,

`keystone.set('file limit', 52428800); // 50mb`

this will solve it. Stick this in your keystone.js (or wherever you're putting your config stuff)

I'm not sure what it's set to by default, but it seems exceedingly low for any file upload usage.

**Comment by christroutner** [comment]
One of my clients just ran into this error when they tried to create a blog post using Keystone. I'm just starting down the path of researching it and figuring out a work around. I'd welcome any tips.

**Comment by christroutner** [comment]
I appear to have fixed the issue I was having. I tried two solutions, and I'm not sure which one was the one that did it.

First, I use nginx to route https stuff on the server. I found this [Stack Overflow thread](https://stackoverflow.com/questions/18121227/how-to-avoid-request-entity-too-large-413-error) that inspired me to add the following line to server block:

`client_max_body_size 50M; #all file uploads up to 50 MB`

I restarted nginx, but I was still getting the error. However, I didn't reboot the server or restart keystone.js.

The second thing I tried was to add this line to keystone.init() in keystone.js:

`'file limit': '50MB'`

I had no idea if this was the right syntax or not. But when I restarted keystone.js, it ran and didn't complain. I rebooted the server and when it came up I stopped getting the entity.to.large error.

Hopefully this helps someone in the future if they come across this issue.
